lib/tapioca/dsl/compilers/active_model_attributes.rb

```diff
@@ -70,7 +70,7 @@ def gather_constants
 
         sig { returns(T::Array[[::String, ::String]]) }
         def attribute_methods_for_constant
-          patterns = if constant.respond_to?(:attribute_method_patterns)
+          patterns = if Helpers::ActiveRecordColumnTypeHelper::HasAttributeMethodPatternsMethod
```

Comment:
I think the helper needs to be required in this file too to access the constant?